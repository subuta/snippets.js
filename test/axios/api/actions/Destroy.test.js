/* global expect, describe, it */

import { build, format, snippets as s } from 'bld.js'

import DestroyAction from 'lib/axios/api/actions/Destroy'

import {
  Routes as RoutesConfig,
  Models as ModelsConfig
} from 'test/fixtures/config'

describe('axios/api/actions/Destroy', () => {
  it('should Destroy axios Api Destroy action with channel', () => {
    const channelRoute = RoutesConfig['channel']
    const channelModel = ModelsConfig['channel']

    expect(format(DestroyAction(channelRoute, channelModel))).toMatchSnapshot()
  })

  it('should Destroy axios Api Destroy action with comment', () => {
    const channelRoute = RoutesConfig['comment']
    const channelModel = ModelsConfig['comment']

    expect(format(DestroyAction(channelRoute, channelModel))).toMatchSnapshot()
  })
})
