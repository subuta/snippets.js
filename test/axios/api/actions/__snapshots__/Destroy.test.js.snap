// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`axios/api/actions/Destroy should Destroy axios Api Destroy action with channel 1`] = `
"export const destroy = (id) => {
  return request.delete(\`/channels/\${id}\`)
}
"
`;

exports[`axios/api/actions/Destroy should Destroy axios Api Destroy action with comment 1`] = `
"export const destroy = (id, params) => {
  return request.delete(\`/channels/\${params.channelId}/comments/\${id}\`)
}
"
`;
